packages/web/src/components/range/MultiRange.js

```diff
@@ -240,16 +240,23 @@ class MultiRange extends Component {
 						{this.props.title}
 					</Title>
 				)}
-				<UL className={getClassName(this.props.innerClass, 'list') || null}>
+				<UL
+					className={getClassName(this.props.innerClass, 'list') || null}
+					role="listbox"
+					aria-label={`${this.props.componentId}-items`}
+				>
 					{this.props.data.map(item => (
 						<li
 							key={item.label}
 							className={`${this.state.selectedValues[item.label] ? 'active' : ''}`}
+							role="option"
+							aria-checked={!!this.state.currentValue[item.label]}
+							aria-selected={!!this.state.currentValue[item.label]}
```

Comment:
Can you keep a variable to store `!!this.state.currentValue[item.label]`, and use it further.

Reply:
@jyash97 Can you update it?